arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,778 @@
+&gpio2 {
+	gpio-line-names = "", "", "", "", "", "", "", "", 
+	                  "", "", "", "", "", "", "", "", 
+	                  "", "", "", "", "", "", "", "", 
+	                  "", "", "", "", "", "", "", ""; 
+};
+
+&gpio3 {
+	gpio-line-names = "HD12_SPI_CS",    "JTAG_FPGA_TCK",  "JTAG_FPGA_TMS",  "JTAG_FPGA_TDI",  
+	                  "WDOG#",          "I2C_3_DAT",      "I2C_3_CLK",      "",               
+	                  "HD12_I2C_CLK",   "HD12_I2C_DAT",   "HD12_DIG_INPUT", "NO_CHRG_JMP#",   
+	                  "EN_NIM_USB#",    "",               "",               "XBEE_CTS#",      
+	                  "U_BOOT_JMP#",    "",               "PUSH_SW_CPU#",   "NIMBEL_PWR_ON",  
+	                  "",               "",               "",               "ID4",            
+	                  "JTAG_FPGA_TDO",  "",               "",               "ID1",            
+	                  "ETH_PHY_RESET#", "",               "",               "";               
+};
+
+&gpio4 {
+	gpio-line-names = "",                "",                "",                "",         
+	                  "",                "",                "SPI_4_CLK",       "SPI_4_MOSI",      
+	                  "SPI_4_MISO",      "SPI_4_CS#",       "MAG_N_IRQ",       "FPGA_RESET#",     
+	                  "SPI_3_FPGA_CS#",  "SPI_3_CLK",       "SPI_3_MOSI",      "SPI_3_MISO",      
+	                  "PWM_5",           "",                "",                "ID5",             
+	                  "GYRO_INT",        "6UL_FORCE_5V_ON", "EN_EMMC_3.3V#",   "EN_YEL_LED#",     
+	                  "EN_RED_LED#",     "EN_GRN_LED#",     "EN_BLU_LED",      "FRAM_SPI_CS#",    
+	                  "SD_VSEL_1.8V",    "",                "",                "";                
+};
+
+&gpio5 {
+	gpio-line-names = "POWER_FAIL",     "GPS_PPS_OUT",    "",               "GPIO_DVFS",      
+	                  "",               "SILAB_C2_CLK",   "SILAB_C2_DATA",  "SILAB_C2_RESET", 
+	                  "SPARE_4",        "",               "",               "",               
+	                  "",               "",               "",               "",               
+	                  "",               "",               "",               "",               
+	                  "",               "",               "",               "",               
+	                  "",               "",               "",               "",               
+	                  "",               "",               "",               "";               
+};
+
+&i2c1 {
+	clock-frequency = <100000>;
+	pinctrl-names = "default", "gpio";
+	pinctrl-0 = <&pinctrl_i2c1>;
+	pinctrl-1 = <&pinctrl_i2c1_gpio>;
+	scl-gpios = <&gpio1 2 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+	sda-gpios = <&gpio1 3 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+	status = "okay";
+
+	/*
+         * Reference: Documentation/devicetree/bindings/iio/st-sensors.txt
+	 *
+	 *   - MAG_N_IRQ enters NAND_ALE/SD2_RST#/PWM3 pad B4
+	 *   - as well as the FPGA
+	 */
```

Comment:
Remove this, not necessary. The reference can be found by looking at the compatible name. If the FPGA pin is unused and otherwise NC then it doesn't need to be mentioned. And the pin itself can be looked up via the pinctrl table.